aca_preview/preview.py

```diff
@@ -325,6 +325,9 @@ def add_review_methods(cls, aca, *, obsid=None, loud=False, preview_dir='.'):
             obj.t_ccd = round(obj.t_ccd, 2)
         aca.acqs.man_angle = round(obj.man_angle, 2)
 
+        # Compute guide count once for the record
+        aca.guide_count = guide_count(aca.guides['mag'], aca.guides.t_ccd, aca.is_ER)
```

Comment:
For ER's we're going to want both `aca.guide_count_9th` and the classic `aca.guide_count`.

Reply:
That makes sense for the checks for me, no problem, but I was hesitant to change any of your stuff.  So I thought the bright ER guide count was already being used instead of guide_count  for the printing and the roll checks.

Reply:
> I thought the bright ER guide count was already being used

It was, and as we saw that was really confusing.